contents/docs/session-replay/installation/web.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Web session replay installation
+platformImageUrl: https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/integrate/js.svg
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+
+export const EnableSessionReplayDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/session-replay/enable-session-replay-in-project-settings-dark-mode.png"
+export const EnableSessionReplayLight = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/session-replay/enable-session-replay-in-project-settings-light-mode.png"
+
+## Step one: Install our JavaScript web library
```

Comment:
I'm not a fan of these Step one Step two things.

Can we just use steps UI? ;)